lxd/storage/drivers/driver_lvm_volumes.go

```diff
@@ -244,6 +244,9 @@ func (d *lvm) HasVolume(vol Volume) bool {
 
 // FillVolumeConfig populate volume with default config.
 func (d *lvm) FillVolumeConfig(vol Volume) error {
+	// Copy volume.* configuration options from pool.
+	d.fillVolumeConfig(vol)
```

Comment:
This will cause problems with different volume types see https://github.com/lxc/lxd/pull/10613/commits/415c89d67db628cb2ff7bcc0862f46e3641138ac#r909604500

Reply:
The same as above.